### DISCUSSION TITLE
Add a function that returns an array of names of all the tilemaps that are present in a certain cell.

### DISCUSSION THREAD

**Discussion by RakibRyan** [godotengine/godot-proposals#9671] [2D]
![Screenshot (31)](https://github.com/godotengine/godot-proposals/assets/85598032/71e2b589-db82-4a82-9b85-1ad76c14d92c)

person more_vertlet's say tilemap1 a has tiles in [1,1],[1,2],[2,1] and tilemap2 has [1,2],[2,2],[2,3] and tilemap3 has [2,2],[2,1]
when I click on let's say [2,2] function outputs an array of all the tilemaps that have a tile in [2,2]. So in this case it'd return [tilemap2,tilemap3]

I was trying to do this but couldn't find a way to do it.

**Comment by AThousandShips** [comment]
If you have multiple `TileMap`s then they aren't necessarily sharing the same positions, this isn't a usual expectation, individual tile maps are individual, this is something you'd have to make yourself for your specific use case IMO

**Reply by RakibRyan** [reply]
there are overlaps. but if i had to could you provide info on how i'd implement such a thing?

**Reply by AThousandShips** [reply]
Just iterate over all your tile maps and call the same function in the scene, something using `find_children("*", "TileMap")`
